CHANGELOG.md

```diff
@@ -1,5 +1,83 @@
+- Bumping `minSdk` level to 19 (Android 4.4)
+- The SDK will now listen to connectivity changes and try to re-upload cached events when internet connection is re-established additionally to uploading events on app restart 
+- `Sentry.getSpan` now returns the root transaction, which should improve the span hierarchy and make it leaner
+- Multiple improvements to reduce probability of the SDK causing ANRs
+- New `sentry-okhttp` artifact is unbundled from Android and can be used in pure JVM-only apps 
+
+**Note: The v7 version of the Android/Java SDK requires a self-hosted version of Sentry 22.12.0 or higher. If you are using a version of [self-hosted Sentry](https://develop.sentry.dev/self-hosted/) (aka onpremise) older than `22.12.0` then you will need to [upgrade](https://develop.sentry.dev/self-hosted/releases/). If you're using `sentry.io` no action needed.**
+
+## Sentry Integrations Version Compatibility
+
+Make sure to align _all_ Sentry dependencies to the same version when bumping the SDK to 7.+, otherwise it will crash at runtime due to binary incompatibility. (E.g. if you're using `-timber`, `-okhttp` or other packages)
+
+For example, if you're using the [Sentry Android Gradle plugin](https://github.com/getsentry/sentry-android-gradle-plugin) with the `autoInstallation` [feature](https://docs.sentry.io/platforms/android/configuration/gradle/#auto-installation) (enabled by default), make sure to use version 4.+ of the gradle plugin together with version 7.+ of the SDK. If you can't do that for some reason, you can specify sentry version via the plugin config block:
+
+```kotlin
+sentry {
+  autoInstallation {
+    sentryVersion.set("7.0.0")
+  }
+}
+```
+
+Similarly, if you have a Sentry SDK (e.g. `sentry-android-core`) dependency on one of your Gradle modules and you're updating it to 7.+, make sure the Gradle plugin is at 4.+ or specify the SDK version as shown in the snippet above.
+
+## Breaking Changes
+
+- Bump min API to 19 ([#2883](https://github.com/getsentry/sentry-java/pull/2883))
+- If you're using `sentry-kotlin-extensions`, it requires `kotlinx-coroutines-core` version `1.6.1` or higher now ([#2838](https://github.com/getsentry/sentry-java/pull/2838))
+- Move enableNdk from SentryOptions to SentryAndroidOptions ([#2793](https://github.com/getsentry/sentry-java/pull/2793))
+- Apollo v2 BeforeSpanCallback now allows returning null ([#2890](https://github.com/getsentry/sentry-java/pull/2890))
+- `SentryOkHttpUtils` was removed from public API as it's been exposed by mistake ([#3005](https://github.com/getsentry/sentry-java/pull/3005))
+
+## Behavioural Changes
+
+- Android only: `Sentry.getSpan()` returns the root span/transaction instead of the latest span ([#2855](https://github.com/getsentry/sentry-java/pull/2855))
+- Capture failed HTTP requests by default ([#2794](https://github.com/getsentry/sentry-java/pull/2794))
```

Comment:
`m`: I feel like we should mention that this can cause increased error volume generated and that ppl can use config options (https://docs.sentry.io/platforms/android/integrations/okhttp#http-client-errors, https://docs.sentry.io/platforms/android/integrations/apollo3#graphql-client-errors) to configure which errors should be reported

Reply:
yeah, good idea, I will update. 

While on it, I realized we didn't flip the flag for the apollo integration, I guess we just missed that. I will also enable it for apollo then